src/components/TodoList.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/jsx-key */
+import React from 'react';
```

Comment:
Eslint guidelines are the list of some common errors and warnings which are identified through the years that caused some problems while working in the group in the development phase or even in the production, so I highly recommend getting familiar with these warnings and following the while, we are learning.